play.py

```diff
@@ -0,0 +1,21 @@
+import player
+import game
+import argparse
+
+parser = argparse.ArgumentParser(description="setup tictactoe game")
+parser.add_argument('b', nargs='?', default=3, type=int, help="board size")
```

Comment:
This would be better as an optional arg `add_argument('-b', '--boardsize', ...)`.

In general anything that has a `default` should be optional (ie have a switch `-` preceding it).

Reply:
(And then it should also have `nargs=1`)